rust/noosphere-sphere/src/sync/gateway.rs

```diff
@@ -231,22 +309,51 @@ where
         println!("Collecting blocks from new local history...");
 
         let bundle = Sphere::at(local_sphere_tip, context.db())
-            .try_bundle_until_ancestor(local_sphere_base.as_ref())
+            .bundle_until_ancestor(local_sphere_base.as_ref())
             .await?;
 
         let client = context.client().await?;
 
+        let local_sphere_identity = context.identity();
+        let authorization = context
+            .author()
+            .require_authorization()?
+            .resolve_ucan(context.db())
+            .await?;
+
+        let name_record = Jwt(UcanBuilder::default()
+            .issued_by(&context.author().key)
+            .for_audience(&local_sphere_identity)
+            .witnessed_by(&authorization)
+            .claiming_capability(&Capability {
+                with: With::Resource {
+                    kind: Resource::Scoped(SphereReference {
+                        did: local_sphere_identity.to_string(),
+                    }),
+                },
+                can: SphereAction::Publish,
+            })
+            .with_lifetime(120)
```

Comment:
Wouldn't this record expire in 2 minutes? We should have some constant, default lifetime duration somewhere

Reply:
SG, although worth noting that at this time we have no expiration of name records (at least, not one that we enforce).

Already filed https://github.com/subconsciousnetwork/noosphere/issues/259 to discuss